src/RingDecomposerLib/CMakeLists.txt

```
@@ -28,4 +28,4 @@ build_library("${SHARED_LIBRARY}" "SHARED")
 build_library("${STATIC_LIBRARY}" "STATIC")
 
 install(FILES RingDecomposerLib.h
-        DESTINATION include)
+        DESTINATION include/RingDecomposerLib)
```

```suggestion
        DESTINATION RingDecomposerLib)
```
I think we can leave this in the main include dir then again because it is only one file.

---

Makes sense.